### ISSUE TITLE
Avails Calendar view - Start calendar as of today

### ISSUE THREAD

**Issue by Jackseed** [blockframes/blockframes#6766]
It's not useful to be able to select dates already passed, they should be in different colour states.

**Comment by 2amarkina** [comment]
Done for UX

**Comment by 2amarkina** [comment]
@stephanie-cascade8 yes, all the cells previous to present month (october in our case) should not be clickable. The only possible action is descriptive tooltip saying: Not available for the passed months (text to confirm). 

@iPR0D1G1 sorry, can't access the link you provided, shows me 404 error. 

To sum up we have 4 states / colors: 

1. All the months (most important rule, goes above the others)  that precede the actual month should be disabled with special color (@stephanie-cascade8 proposed disabled color, light grey). 
2. If rights are sold, then use sold color (dark grey) 
3. If available bright blue
4. If already selected other blue  

So on the last screen case we need to have the light grey color going from January till September (included). 
If we do not have any information about these previous months that's still ok to disable them, because they have already passed and no one will buy rights for the months that passed.

**Comment by iPR0D1G1** [comment]
yes, that's the current implementation.
